### ISSUE TITLE
Review 527EZ ITF alerts

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#79641]

**Comment by katherine-fung** [comment]
Hey @aprocik1, 

I've drafted some copy for the intent to file alert in the pension form. For context, the Supplemental Claim and disability claim forms also have intent to file alerts. 

The copy I'm proposing is different from both of these. I use the term "intent to file" less often (aka not in the header) because Gretchen noted that many Veterans seem unfamiliar with the term. (notes here: https://www.figma.com/file/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?type=design&node-id=2181-5007&mode=design&t=u9nQpxFUT7KQSDml-0) 

I'm also debating whether we need to add a link at the end [[Learn more about intent to file (opens in new tab)](https://www.va.gov/resources/your-intent-to-file-a-va-claim/)], which will take Veterans away from the form flow. 

also cc'ing @laurwill for awareness of the differences between the ITF alerts

> [header options] 
> We've recorded your application start date
> We've recorded a potential start date for your benefits 
> We've recorded your intent to file 
> 
> When you start an application for pension benefits online, we record a potential start date for your benefits [_note: this sentence could potentially be confusing with a header that says "application start date"_]. This is called your intent to file. 
> 
> If we approve your claim, you may be able to get retroactive payments (payments for the time between when you started your application and when we approve your application). 
> 
> You must complete and submit your application by **Thursday, March 6th, 2025 at 12:07 p.m. ET.** The potential start date for your benefits resets one year after you start your application. 
> 
> You don't need to do anything else to notify us of your intent to file. You can continue to fill out your application. 

[I will drop in screenshots of the disability and Supplemental Claims ITF alerts]

**Comment by aprocik1** [comment]
@katherine-fung, thanks! I don't think we need to link to the [Your intent to file a VA claim](https://www.va.gov/resources/your-intent-to-file-a-va-claim/) page. I don't think we should take users out of the form flow, especially because the page doesn't include much more information than the alert.

What do you think about this? I'm trying to simplify the mention of "retroactive payments."

>[Header] We've recorded a potential start date for your benefits
>
>When you start an application for pension benefits online, we record a potential start date for your benefits. We call this your intent to file.
>
>You may be able to get payments for the time between your potential start date and the date we approve your application.
>
>You must submit your application by **Thursday, March 6th, 2025 at 12:07 p.m. ET** to keep your potential start date. Your potential start date resets one year after you start your application.
>
>You don't need to do anything else right now. You can continue to fill out your application.

**Comment by katherine-fung** [comment]
Thanks @aprocik1 - your edits are great. 

One question: do you think we should tweak to say, "When you start an application for pension benefits online, we record **_the date as the_** potential start date for your benefits. We call this your intent to file."

**Comment by aprocik1** [comment]
Good call! What do you think about either of these? I'm trying to explicitly state what we mean by "the date."

>We record the date that you start an application for pension benefits online as the potential start date for your benefits. And we call this date your intent to file.

>If you apply for pension benefits online, we record the date that you start your application as the potential start date for your benefits. We call this date your intent to file.

**Comment by katherine-fung** [comment]
@aprocik1, what do you think of these minor tweaks in bold?  

> [Header] We've recorded a potential start date for your benefits
> 
> **When** you apply for pension benefits online, we record the date that you start your application as the potential start date for your benefits. **We call this your intent to file.** or **We automatically receive your intent to file.** 
> 
> You may be able to get payments for the time between your potential start date and the date we approve your application.
> 
> You must submit your application by Thursday, March 6th, 2025 at 12:07 p.m. ET to keep your potential start date. Your potential start date resets one year after you start your application.
> 
> You don't need to do anything else right now. You can continue to fill out your application.

**Comment by aprocik1** [comment]
@katherine-fung, this is a great edit. Thanks for working on this!

> [Header] We've recorded a potential start date for your benefits
> 
> When you apply for pension benefits online, we automatically record your intent to file. This means we record the date you start your application as the potential start date for your benefits.
> 
> You may be able to get payments for the time between your potential start date and the date we approve your application.
> 
> You must submit your application by **Thursday, March 6th, 2025 at 12:07 p.m. ET** to keep your potential start date. Your potential start date resets one year after you start your application.
> 
> You don't need to do anything else right now. You can continue to fill out your application.

**Comment by katherine-fung** [comment]
Thanks @aprocik1! I passed the rec along to Gretchen in ticket #78928. Leaving this ticket open in case her team has questions.

**Comment by katherine-fung** [comment]
Hi @aprocik1, 

The product team is currently discussing how many states they need for this alert, and the technical behavior of the ITF in the 527 form. 

Based on what I know so far, I'm drafting 2 states (1. state where the user has an ITF, 2. state where the system can't return info about the ITF whether they are starting their form and don't have an ITF yet, or they started their form in the past and the system just can't access the ITF info right now). 

**1. state where the user has an ITF**

> [Header] We've recorded a potential start date for your benefits
> 
> When you apply for pension benefits online, we automatically record your intent to file. This means we record the date you start your application as the potential start date for your benefits.
> 
> If we approve your application, the potential start date for your benefits is **Wednesday, March 6, 2024**. 
> 
> You may be able to get payments for the time between this date and the date we approve your application.
> 
> You must submit your application by Thursday, March 6, 2025 at 12:07 p.m. ET to keep your potential start date. Your potential start date resets one year after you start your application.
> 
> You don't need to do anything else right now. You can continue to fill out your application.

2. state where the system can't return info about the ITF whether they are starting their form and don't have an ITF yet, or they started their form in the past and the system just can't access the ITF info right now). I'm leveraging the [copy from the 526 alert](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/gh/department-of-veterans-affairs/va.gov-team/71506). The 527 team needs to confirm what happens in the backend before I know whether this copy is accurate. 

> [H2] We're confirming the potential start date for your benefits 
> 
> When you apply for pension benefits online, we automatically record your intent to file. This means we record the date you start your application as the potential start date for your benefits.
> 
> If you're returning to a saved application, we'll continue to hold the date you started your application as the potential start date for your benefits.
> 
> If you're starting a new application, we'll use today as the potential start date for your benefits.
> 
> You don't need to do anything else right now. You can continue to fill out your application.

**Comment by katherine-fung** [comment]
@aprocik1 @strelichl 

[Per Slack](https://dsva.slack.com/archives/C06E83RNLQM/p1713896788375049?thread_ts=1713214661.040629&cid=C06E83RNLQM), I'm holding on this work until the product team circles back and confirms with us the business requirements for the alert (including 1. what states of the alert they require, and 2. what information they need these alerts to provide to the user).

**Comment by katherine-fung** [comment]
Noting that @echin393 and I spoke about the possible states of the ITF alert and what may happen in the backend for these states. She says that the team is still discussing what happens when a user returns to the form and their ITF has expired. She'll keep me posted about what they decide. When this is confirmed, she'll let me know and I'll return to this copy to refine it. I also let her know that I've drafted copy in the comment above. 

cc @aprocik1 @strelichl @juliepedtke

**Comment by katherine-fung** [comment]
I've delivered content recs for the ITF alerts in the team's Mural.

**Comment by aprocik1** [comment]
@strelichl, you can close this work ticket, but don't close the intake ticket yet. The product team may ask for more content support after their research and testing. Thanks!
